tools/yaffs2/Makefile

```diff
@@ -10,12 +10,12 @@ include $(TOPDIR)/rules.mk
 PKG_NAME:=yaffs2_android
 PKG_VERSION:=2008-12-18
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://android.googlesource.com/platform/external/yaffs2
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)
 PKG_SOURCE_VERSION:=d333fc232d7e5ae3370080d5d6f7d88ea9c6b3a1
-PKG_MIRROR_MD5SUM:=8145d4dd9b46face26121f27ad60cc1c
+PKG_MIRROR_MD5SUM:=
 HOST_BUILD_DIR=$(BUILD_DIR_HOST)/$(PKG_NAME)
```

Comment:
It's an issue because git doesn't date and time-stamp files during checkout so each package will be different unless you checkout at the exact same time and at the exact same speed. As someone who doesn't have access to the "fallback" download server it's impossible/doesn't make any sense to add a checksum for a package that's more than likely mismatch what's going to be uploaded if any at all. There are more Makefiles in the tree which have empty PKG_MIRROR_MD5SUM vars so I figured it would be more consistent to leave those empty rather than removing them.